src/content/guides/compatible-terminals.mdx

```diff
@@ -0,0 +1,149 @@
+---
+title: Compatible Terminals
+description: Compatible payment terminals.
+nav:
+  path: Reference/Merchant Integrations
+  order: 1
+---
+import Disclosure from '../../components/Disclosure.vue';
+import Faq from '../../components/Faq.astro';
+
+The Centrapay service is supported on a range payment and point of sale devices listed below. If the device model is not in this list, it is not supported. If you would like to discuss a new integration or support of Centrapay on a new device, please contact Centrapay at support@centrapay.com
+
+## Terminal Providers
+
+ <Faq>
```

Comment:
@Wulingoh Suggestion: This component is not being used for FAQs - it needs a more generic name

Reply:
@tom-blake Unfortunately, this is how the component is name. When I removed the Faq component, it removes the styling. I would have to create another component with similar css.

Reply:
@Wulingoh Couldn't you rename the component?

Reply:
@tom-blake Shall I name it as `AccordionListWrapper`